tests/framework/AnalyticModels/tensor_poly2.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#***************************************
+#* Simple analytic test ExternalModule *
+#***************************************
+#
+#  This is a simple polynomial evaluation of all single-order combination polynomials
+#    For instance, xyz + xy + xz + yz + x + y + z + 1
+#
+import numpy as np
```

Comment:
what is the different between this file and the existing tenso_poly file?

Reply:
The output variable name is 'ans2' instead of 'ans' in tensor_poly file. I need to use output values of two different external models, whose name are both 'ans' in their model file, I don't know how to specify two variables with the same name in my .xml file...

Reply:
I changed the ExternalModel to projectile.py

Reply:
This file is not in use in current test.xml